python/lsst/ip/isr/isrFunctions.py

```diff
@@ -216,6 +217,108 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         fpSet.setMask(mask, maskValue)
 
 
+def maskITLEdgeBleed(ccdExposure, itlEdgeBleedSatMinArea,
+                     itlEdgeBleedSatFracLevel, itlEdgeBleedModelConstant, saturatedMaskName, badAmpDict):
+    """Mask edge bleeds in ITL detectors.
+
+    Parameters
+    ----------
+    ccdExposure : `lsst.afw.image.Exposure`
+        Exposure to apply masking to.
+    itlEdgeBleedSatMinArea : scalar
+        Minimal saturated footprint area where the presence of edge bleeds
+        will be checked.
+    itlEdgeBleedSatFracLevel : scalar
+       Fraction of the saturation level at the detector edge
+       above which there is an edge bleed.
+    itlEdgeBleedModelConstant : scalar
+        Constant in the decaying exponential in the edge bleed masking.
+    saturatedMaskName : `str`
+        Mask name for saturation.
+    badAmpDict : `str`[`bool`]
+        Dictionary of amplifiers, keyed by name, value is True if
+        amplifier is fully masked.
+    """
+    maskedImage = ccdExposure.maskedImage
+
+    # Get minimum of amplifier saturation level
+    satLevel = numpy.min([ccdExposure.metadata[f"LSST ISR SATURATION LEVEL {amp.getName()}"]
+                          for amp in ccdExposure.getDetector() if not badAmpDict[amp.getName()]])
+
+    # Get footprint list of saturated pixels
+    thresh = afwDetection.Threshold(satLevel)
+    fpList = afwDetection.FootprintSet(maskedImage, thresh).getFootprints()
+
+    satAreas = numpy.asarray([fp.getArea() for fp in fpList])
+    largeAreas, = numpy.where(satAreas >= itlEdgeBleedSatMinArea)
+
+    if len(largeAreas) == 0:
+        return
+    else:
```

Comment:
No need for an `else` here.  The below can be shifted one level left, the other check just kicks back out anyway.